moveit_ros/hybrid_planning/hybrid_planning_manager/hybrid_planning_manager_component/include/moveit/hybrid_planning_manager/hybrid_planning_manager.h

```diff
@@ -76,7 +76,12 @@ class HybridPlanningManager : public rclcpp::Node
    */
   std::shared_ptr<HybridPlanningManager> shared_from_this()
   {
-    return std::static_pointer_cast<HybridPlanningManager>(Node::shared_from_this());
+    return std::static_pointer_cast<HybridPlanningManager>(shared_from_this());
+  }
```

Comment:
Since now the class inherits from `std::enable_shared_from_this<HybridPlanningManager>`, do we need this function?